### PR TITLE
Use Entity.to_json method of Core

### PR DESCRIPTION
**Pull request by cbenz**
Depends on with https://github.com/openfisca/openfisca-core/pull/425

### CODE REVIEW

File: openfisca_web_api/controllers/entities.py [hunk 1]
```diff
@@ -9,19 +9,8 @@
...
-    else:
-        entity_data.update({
-            'roles': entity.roles_description,
-            })
-    return entity_data
-
-
 @wsgihelpers.wsgify
-def api1_entities(req):
+def api2_entities(req):
```
Comment (MattiSG) [inline openfisca_web_api/controllers/entities.py]: Why rename if you don't keep the previous version too?

Comment (cbenz) [inline openfisca_web_api/controllers/entities.py]: Hmm, this is to follow the naming in use (`/api/N/name` => `apiN_name`): see [`controllers/__init__.py`](https://github.com/openfisca/openfisca-web-api/blob/c7a16ff786c1869a9e7bc7aed142c8f4b6cd0b7a/openfisca_web_api/controllers/__init__.py#L37-L48).

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+--editable git+https://github.com/openfisca/openfisca-core.git@entity-to-json#egg=OpenFisca-Core
```
Comment (fpagnoux) [inline requirements.txt]: This is probably not something we want on master.

Comment (cbenz) [inline requirements.txt]: This is required for tests to pass, while https://github.com/openfisca/openfisca-core/pull/425 is not merged. I remove it, and let's wait until 425 is merged.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.2.2
+
+* Use `Entity.to_json` method (see [commit on Core](https://github.com/openfisca/openfisca-core/commit/dd4d5df3b16ccdd77d5afaece608b13812484e9c))
```
Comment (fpagnoux) [inline CHANGELOG.md]: Changelogs should be less technical (I can see we are using the `to_json` method in the code) and more functional. What is bringing this change to the api ? Is it repairing a broken route ? Are we changing the JSON representation (and, if so, is it a breaking change ?).

**Comment by fpagnoux** [comment]
There is also some 12 days old requested changes from @MattiSG. If the review has been treated, let's ask him a new one.

**Comment by cbenz** [comment]
Sorry for rebasing and forcing the push, but there was a conflict with `CHANGELOG.md` and `setup.py`

File: openfisca_web_api/controllers/entities.py (hunk 1 above)
Comment (MattiSG) [inline openfisca_web_api/controllers/entities.py]: Judging from the route changes that were added after this comment, you mean the `v1` of `entities` had already been removed? Ok, then renaming was needed.

**Pull request merged**